datalad/support/sshconnector.py

```diff
@@ -55,20 +56,22 @@ def get_connection_hash(hostname, port='', username='', identity_file=''):
     #  https://github.com/ansible/ansible/issues/11536#issuecomment-153030743
     #  https://github.com/datalad/datalad/pull/1377
     return md5(
-        '{lhost}{rhost}{port}{identity_file}{username}'.format(
+        '{lhost}{rhost}{port}{identity_file}{username}{force_ip}'.format(
             lhost=gethostname(),
             rhost=hostname,
             port=port,
             identity_file=identity_file,
-            username=username).encode('utf-8')).hexdigest()[:8]
+            username=username,
+            force_ip=force_ip
```

Comment:
I would have for consistency had `force_ip=False` in the signature of this function as well, and then do `or ''` right here?

Reply:
> I would have for consistency had `force_ip=False` in the signature of this function as well, and then do `or ''` right here?

Given the context, I wrote it how I did to be consistent with the other arguments.  If I were writing the function from scratch, I'd have written it differently.  But I must admit that I don't think this matters at all.  I'm guessing you do since it elicited a comment, so I'll change it to the form you prefer.